Source/WebCore/rendering/svg/SVGRenderingContext.cpp

```diff
@@ -299,4 +299,32 @@ bool SVGRenderingContext::bufferForeground(RefPtr<ImageBuffer>& imageBuffer)
     return true;
 }
 
+RefPtr<ImageBuffer> SVGRenderingContext::createImageBuffer(const FloatRect& targetRect, const AffineTransform& absoluteTransform, const DestinationColorSpace& colorSpace, RenderingMode, const GraphicsContext* context)
```

Comment:
Do we really want to add stuff to SVGRenderingContext? I am trying to make SVGRenderSupport / SVGRenderingContext legacy-only.

Reply:
Sure, removed.